tests/test_multimodal_preprocessing.py

```diff
@@ -0,0 +1,88 @@
+"""
+Test cases for multimodal preprocessing in tcga_utilities.py
+"""
+import string
+import itertools as it
+
+import pytest
+import numpy as np
+import pandas as pd
+
+import mpmp.config as cfg
+import mpmp.utilities.tcga_utilities as tu
+
+@pytest.fixture(scope='module')
+def raw_data():
+    """Generate a raw dataset"""
+    cols = list(string.ascii_lowercase)[:11]
+    X_train_raw_df = pd.DataFrame(np.random.uniform(size=(20, 11)), columns=cols)
+    X_test_raw_df = pd.DataFrame(np.random.uniform(size=(10, 11)), columns=cols)
+    data_types = np.array([0, 0, 0, 1, 1, 1, 2, 2, 2, -1, -1])
```

Comment:
What I have now doesn't use a fixed index for each data type - it just assigns them in the order the data types are passed in. So if you wanted to build a model from expression data and 27K methylation data, it would assign those data types indexes 0 and 1 in that order.

I think I'll keep it the way it is since it's flexible (if I add other data types in the future it should generalize automatically), but I do like your idea of having a constant for non-gene features. I'll add that.